<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_rpt_report_frame_dishquit</class>
 <widget class="QWidget" name="w_rpt_report_frame_dishquit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>49</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>单号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_lineedit" name="lineEdit_billno"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_uncheck">
     <property name="text">
      <string>反结账</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_lineedit</class>
   <extends>QLineEdit</extends>
   <header>lds_lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
